pkgs/misc/drivers/xow/default.nix

```diff
@@ -0,0 +1,29 @@
+{ stdenv, fetchurl, libusb, gitMinimal }:
+
+stdenv.mkDerivation rec {
+  pname = "xow";
+  version = "0.2";
+
+  src = fetchurl {
+    url = "https://github.com/medusalix/xow/archive/v${version}.tar.gz";
```

Comment:
You can use `fetchFromGitHub` here instead (make sure to check that the `sha256` is still correct then)

Reply:
Fixed.